AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -8210,7 +8210,7 @@ humboldtforum.org#%#//scriptlet('set-cookie', 'acceptCookie', 'true')
 hometogo.de#%#(function(){document.cookie.includes("cookie_consent=")||(document.cookie='cookie_consent={"timestamp":'+Date.now()+',"functional":true,"analytical":false,"marketing":false,"names":{"localStorage":{"analyticsdata":0,"analyticslastAction":0},"cookie":{"bftul":0},"thirdParty":{"facebookTracking":0,"hotJar":0,"trustBadge":0,"braze":0,"gtm_a_adup":0,"gtm_a_bing":0,"gtm_a_ctri":0,"gtm_a_fb":0,"gtm_a_ga":0,"gtm_a_gads":0,"gtm_a_pin":0,"gtm_a_tvsq":0,"gtm_a_yand":0,"gtm_m_adup":0,"gtm_m_crit":0,"gtm_m_gads":0,"gtm_m_myta":0,"gtm_m_rtbh":0,"gtm_m_tabo":0,"gtm_a_tiktok":0,"adcellTracking":0,"tradetrackerTracking":0}}}')})();
 allmusicitalia.it#%#//scriptlet("set-constant", "initCookieConsent", "noopFunc")
 cam.start.canon#%#//scriptlet('set-cookie', 'user_accepts_cookies', 'false')
-bike24.de#%#(function(){document.cookie.includes("cookieConsent=")||(document.cookie='cookieConsent={"basket-store":true,"emarsys":false,"google-ga":false,"google-tm":false,"trbo":false}; path=/;')})();
+bike24.it,bike24.fr,bike24.at,bike24.es,bike24.com,bike24.de#%#(function(){document.cookie.includes("cookieConsent=")||(document.cookie='cookieConsent={"basket-store":true,"emarsys":false,"google-ga":false,"google-tm":false,"trbo":false}; path=/;')})();
```

Comment:
You need just use `bike24.*` rule, `*` can blocks other subdomains too.
You can change the rule like that: `bike24.*#%#(function(){document.cookie.includes("cookieConsent=")||(document.cookie='cookieConsent={"basket-store":true,"emarsys":false,"google-ga":false,"google-tm":false,"trbo":false}; path=/;')})();`